test/masq.test.js

```diff
@@ -555,3 +578,50 @@ test('should set a watcher', async (done) => {
   masq.watch('/hello', onChange)
   await masq.put(key, value)
 })
+
+test('should be able to get a notif on change in masq-app with a watcher on masq-lib', async (done) => {
+  expect.assertions(2)
+  let resolvePr1
+  const pr1 = new Promise((resolve) => { resolvePr1 = resolve })
+  let resolvePr2
+  const pr2 = new Promise((resolve) => { resolvePr2 = resolve })
+
+  const onChangeMasqLib = () => {
+    expect(true).toBe(true)
+    resolvePr1()
+  }
+  const onChangeMockMasqApp = () => {
+    expect(true).toBe(true)
+    resolvePr2()
+  }
+
+  await logInWithMasqAppMock()
+  const key = '/hello'
+  const value = { data: 'world' }
+  mockMasqApp.watch(masq.userId, '/hello', onChangeMockMasqApp)
+  masq.watch('/hello', onChangeMasqLib)
+  await mockMasqApp.put(masq.userId, key, value)
+
+  await Promise.all([pr1, pr2])
+  done()
+})
+
+test('put/get should put an item and get in Mock Masq App', async () => {
```

Comment:
I'm not sure what you want to test here, why do we have a watch called ?

Reply:
I guess you want to wait the data to be syncronized. In this case, can you create a new test file, inside a `describe` called `replication` or something like this

Reply:
I want to test the replication.
We have to wait before getting the value to make sure the 2 hyperdb instances are synced after the put.

We could put the last two tests in a separate file but I don't see the point as we have not isolated the other tests. Maybe we should separate tests into login, get/put/delete, watch and replication ?
But I think we can leave them in the same file for now, what do you think ?

Reply:
Okay !

Reply:
done